app/models.py

```diff
@@ -326,11 +326,11 @@ class Domain(db.Model):
 
 
 ORGANISATION_TYPES = [
-    "central", "local", "nhs_central", "nhs_local", "emergency_service", "school_or_college", "other",
+    "central", "local", "nhs_central", "nhs_local", "gp", "emergency_service", "school_or_college", "other",
```

Comment:
hmm... I feel it might be more useful to call the new org type `nhs_gp`, then if we look for all things 'nhs' it will come up. What do you think?

Reply:
@CrystalPea that’s a good suggestions, I‘ve changed it in 758fbdb